frontend/src/features/leases/detail/LeasePages/deposits/components/DepositsReceivedContainer/DepositsReceivedContainer.tsx

```diff
@@ -0,0 +1,48 @@
+import { FormSection } from 'components/common/form/styles';
+import { Table } from 'components/Table';
+import { ILeaseSecurityDeposit, ILeaseSecurityDepositReturn } from 'interfaces';
+import { Button } from 'react-bootstrap';
+
+import * as Styled from '../../styles';
+import { DepositListEntry, getColumns } from './columns';
+
+export interface IDepositsReceivedContainerProps {
+  securityDeposits: ILeaseSecurityDeposit[];
+  depositReturns: ILeaseSecurityDepositReturn[];
+  onAdd: () => void;
+  onEdit: (id: number) => void;
+  onDelete: (id: number) => void;
+  onReturn: (id: number) => void;
+}
+
+const DepositsReceivedContainer: React.FC<IDepositsReceivedContainerProps> = ({
+  securityDeposits,
+  depositReturns,
+  onAdd,
+  onEdit,
+  onDelete,
+  onReturn,
+}) => {
+  const columns = getColumns({ onEdit, onDelete, onReturn });
+  const dataSource = securityDeposits.map<DepositListEntry>(d => {
+    return new DepositListEntry(d, depositReturns.filter(r => r.parentDepositId === d.id).length);
+  });
+  return (
+    <FormSection>
+      <Styled.SectiontHeader>Deposits Received</Styled.SectiontHeader>
```

Comment:
SectiontHeader ? or just SectionHeader

Reply:
Good find, It looks like it was misspelled in several places